openmetadata-ui/src/main/resources/ui/src/locale/languages/fr-fr.json

```diff
@@ -473,24 +506,31 @@
     "service-email-required": "Un Email pour le Service account est requis",
     "special-character-not-allowed": "Les charactères spéciaux ne sont pas authorisés",
     "sql-query-tooltip": "Requête avec 1 ligne ou plus entrainera l'échec du test.",
+    "stage-file-location-message": "Temporary file name to store the query logs before processing. Absolute file path required.",
     "successfully-completed-the-tour": "Vous avez fini la visite avec succès.",
     "sure-to-remove": "Etes-vous sûr de vouloir supprimer le",
     "team-no-asset": "Votre equipe n'a pas de resources de données",
+    "thread-count-message": "Set the number of threads to use when computing the metrics. If left blank, it will default to 5.",
     "token-end-point-required": "TokenEndpoint est requis",
     "token-expiration-required": "L'Expiration du Jeton est requis",
     "token-security-description": "Toutes personnes en possession de votre Jeton JWT pourra envoyer des requêtes a l'API REST d'OpenMetadata. Ne surtout pas rendre public le Jeton JWT dans le code de votre application. Ne le partagez pas sur Github ou ailleur en ligne.",
     "total-entity-insight": "Montre le nombre le plus récent de resources de données par type.",
+    "use-fqn-for-filtering-message": "Regex will be applied on fully qualified name (e.g service_name.db_name.schema_name.table_name) instead of raw name (e.g. table_name).",
     "view-deleted-teams": "Voir toutes les équipes supprimées qui relèvent de cette équipe.",
     "webhook-listing-message": "Le webhook permet aux services externes d'être notifiés lors de changements dans les métadonnées au seinn de votre organization et cela grâce a l'API . Enregistrez l'URL de redirection (callback URLs) avec l'intégration webhook pour recevoir des notifications lors de changements dans les métadonnées. Vous pouvez ajouter, \"list\", \"update\", and \"delete\" these \"webhooks\".",
     "webhook-type-listing-message": "Envoyez des mises à jour à temps aux producteurs et consommateurs de métadonnées grâce aux notifiactions {{webhookType}}. Utilisez le webhook {{webhookType}} pour envoyer des notifications sur les changements dans les métadonnées dans votre organization grâce a l'API. Vous pouvez ajouter, \"list\", \"update\", and \"delete\" these \"webhooks\".",
     "would-like-to-start-adding-some": "Désirez commencer à ajouter des ",
     "write-your-description": "Ecriver votre description"
   },
   "server": {
+    "add-feed-error": "Error while adding feed!",
```

Comment:
How did we translated it?

Reply:
I have just added keys in fr-fr.json, so we don't miss out the keys in it